seeds/02-posts.js

```diff
@@ -0,0 +1,12 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('posts').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('posts').insert([
+        {id: 1, user_id: 'rowValue1', question:'', answer:'', likes:'', comments:'', track:'', category:'', created_at:'', updated_at:''},
```

Comment:
For these inserts, I think you should enter some actual dummy data for the front-end to work with, so provide data for fields like question and user_id. created_at and updated_at should also be removed since they're updated automatically.

Reply:
Sounds good currently working on the update!

Reply:
updated